src/main/java/frc/robot/subsystems/elevator/commands/Height.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.elevator.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.elevator.Elevator;
+
+public class Height extends CommandBase {
+    Elevator elevator;
+    double height;
+
+    public Height(Elevator elevator, double height) {
+        this.height = height;
+        this.elevator = elevator;
+        addRequirements(elevator);
+    }
+
+    @Override
+    public void initialize() {
+        elevator.setHeight(height);
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        elevator.stop();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return Math.abs( elevator.getheight() - height);
+    }
+}

```

Comment:
A few unrelated notes:
1. `isFinished()` need to return a `boolean`, but `Math.abs` returns a `double` in this case.
2. [`TalonFX.getClosedLoopError()`](https://api.ctr-electronics.com/phoenix/release/java/com/ctre/phoenix/motorcontrol/IMotorController.html#getClosedLoopError(int)) can quite help you here 🙂.
2. `isFinished()` should come before `end()`.